app/qml/ExternalResourceBundle.qml

```diff
@@ -29,6 +29,12 @@ Item {
          */
         property var capturePhoto: function capturePhoto(itemWidget) {
           externalResourceHandler.itemWidget = itemWidget
+          if ( !__inputUtils.createDirectory( itemWidget.targetDir ) )
+          {
+            errorDialog.errorText = qsTr( "Could not create directory %1. Please ensure that your project configured correctly" ).arg( itemWidget.targetDir )
```

Comment:
I would remove "Please ensure that your project configured correctly". Also make sure it is logged in console log too!